### PR TITLE
update content with new repository and CentOS7

### PR DESCRIPTION
**Pull request by lgcosta**
Add instructions to use the new repository adn update instructions to install packages.
Tested installation based on documentation.
- Centos 6
- Centos 7
- Debian Jessie
- Debian Stretch
- Ubuntu 14.04
- Ubuntu 16.04

### CODE REVIEW

File: doc/installation/index.rst [hunk 1]
```diff
@@ -4,8 +4,8 @@
 Installation
 ############
 
-Centreon recommends using its official packages from the Centreon
-Entreprise Server (CES) repository. Most of Centreon' endorsed
+Centreon recommends using its official packages from the Centreon Open Sources
+version available free of charge on our repository (ex CES). Most of Centreon endorsed
```
Comment (bouda1) [inline doc/installation/index.rst]: COS instead of CES ??

File: doc/installation/index.rst [hunk 2]
```diff
@@ -15,29 +15,42 @@ software by following the :ref:`user_installation_using_sources`.
...
 
+
+CentOS 7
+--------
+
+Run the following commands as privileged user ::
+
+  $ wget http://yum.centreon.com/standard/3.4/el7/stable/noarch/RPMS/centreon-release-3.4-4.el7.centos.noarch.rpm
+  $ yum install --nogpgcheck -y centreon-release-3.4-4.el7.centos.noarch.rpm
+  $ rm -f centreon-release-3.4-4.el6.noarch.rpm
```
Comment (bouda1) [inline doc/installation/index.rst]: el7, not el6

File: doc/installation/index.rst (hunk 1 above)
Comment (lgcosta) [reply doc/installation/index.rst]: I will remove "(ex CES)", is based in another documentantion in Centreon-web. Thanks

File: doc/installation/index.rst (hunk 2 above)
Comment (lgcosta) [reply doc/installation/index.rst]: ops ! Thanks :-)

File: doc/installation/index.rst [hunk 3]
```diff
@@ -15,29 +15,42 @@ software by following the :ref:`user_installation_using_sources`.
...
 **************
 
-Centreon provides RPM for its products through Centreon Entreprise
-Server (CES). Open source products are freely available from our
-repository. These packages are available for CentOS 6 x86_64, which
-is the sole platform officially supported by Centreon
+Centreon provides RPM for its products through Centreon Open Sources (COS) version
+available free of charge on our repository.
+Open source products are freely available from our repository.
+These packages are available for CentOS 6 or 7 architeture x86_64, which is
```
Comment (ganoze) [inline doc/installation/index.rst]: ... archite**C**ture x86_64, which are the sole platforms ...

**Comment by lgcosta** [comment]
done. :smiley: 
Thank you

**Comment by ganoze** [comment]
Thank _you_ :-)

**Pull request merged**